tracee-ebpf/tracee/tracee.go

```diff
@@ -1108,6 +1155,40 @@ func (t *Tracee) processEvent(ctx *context, args map[argTag]interface{}) error {
 	return nil
 }
 
+func getFileHash(fileName string) string {
+	f, _ := os.Open(fileName)
+	if f != nil {
+		defer f.Close()
+		h := sha256.New()
+		_, _ = io.Copy(h, f)
+		return hex.EncodeToString(h.Sum(nil))
+	}
+	return ""
+}
+
+func (t *Tracee) updateProfile(sourceFilePath string, executionTs uint64) {
+	if pf, ok := t.profiledFiles[sourceFilePath]; !ok {
+		t.profiledFiles[sourceFilePath] = profilerInfo{
+			Times:            1,
+			FirstExecutionTs: executionTs,
```

Comment:
won't this break the profile? meaning we it won't be comparable to previous run

Reply:
Good point but no, it's explicitly filtered out in the JSON that's saved https://github.com/aquasecurity/tracee/pull/725/files#diff-710a36e0ca7a07ea49cf4970d916ed0b74b1c3619920145d9db50477c4f96967R183

Reply:
A simplification suggestion (not for this PR):
The ctx.Ts is included in the captured file name in order to distinguish between different captured files with same path name.
IIUC you track the FirstExecutionTs field in order to reconstruct the same file name when calculating the hash.
Suggestion is to replace the ctx.Ts in captured file names with the file's ctime (it makes more sense regardless of this PR even), and then the unique id of a captured/profiled file includes just the mntid, path, and ctime. the profile map from destinationFilePath to executions counter and when you calculate the hash you can just iterate over the map/files without reconstructing file names which might be error prone (and also simplify the code flow as no need to track ctx.Ts in the map and pass it around).
Let merge this PR and maybe discuss this in the future